lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -371,17 +362,6 @@ private void init(@Nullable AttributeSet attrs, @AttrRes int defStyleAttr) {
     }
   }
 
-  /**
-   * Allows ignoring system animations settings, therefore allowing animations to run even if they are disabled.
-   * <p>
-   * Defaults to false.
-   *
-   * @param ignore if true animations will run even when they are disabled in the system settings.
-   */
-  public void setIgnoreDisabledSystemAnimations(boolean ignore) {
```

Comment:
Removing an existing public API is tricky. At the very least, we'll need to deprecate this and provide a graceful fallback. Ideally, its behavior would remain the same (scope IgnoreDisabledSystemAnimations() to this view if possible).

It can be fully removed after a few releases.

Reply:
@gpeal make sense, added the original API back.